app/src/main/java/com/cmput301f20t21/bookfriends/entities/Book.java

```diff
@@ -58,19 +56,10 @@ public String getOwner() {
         return owner;
     }
 
-    public BOOK_STATUS getBookStatus() {
+    public BOOK_STATUS getStatus() {
```

Comment:
has to be `getStatus` or it' won't be a simple getter matching status attribute and toObject just cannot convert it.